src/components/ContentSidebar/SidebarContent.scss

```diff
@@ -24,7 +24,7 @@
         top: 50px; // Same as title above
 
         .bcs-scroll-content {
-            width: 325px; // 15px less than the total width for scroll bar
```

Comment:
@priyajeet 
Removing both hard-coded widths fixes the problem as well.

Reply:
This has to remain, it will break other sidebars.